### ISSUE TITLE
xpytorch + paddlepaddle-xpu同时import会出现兼容性错误，且为paddle报错

### ISSUE THREAD

**Issue by sungenglab** [PaddlePaddle/Paddle#72089]
### bug描述 Describe the Bug

<img width="978" alt="Image" src="https://github.com/user-attachments/assets/1f266a56-f20b-4eae-a7c6-41784af2ef5f" />

### 其他补充信息 Additional Supplementary Information

import numpy as np
import paddle
import torch

np.random.seed(42)

shape = (3, 3)
input_numpy = np.random.randn(*shape)

torch.device("xpu:0")
input_torch = torch.tensor(input_numpy, dtype=torch.float32)
out_torch = input_torch.mean()
print(out_torch, "torch input mean")

paddle.device.set_device("xpu")
input_paddle = paddle.to_tensor(input_numpy, dtype=paddle.float32)
out_paddle = paddle.mean(input_paddle)
print(out_paddle, "paddle input mean")

**Comment by sungenglab** [comment]
import paddle
paddle.utils.run_check()
会返回以下错误：

<img width="947" alt="Image" src="https://github.com/user-attachments/assets/9950d580-0c06-41dd-bb59-4c8d37b51dba" />

**Comment by sungenglab** [comment]
use paddle version 3.0.0
5beb46d278a911a2055ae709e61e5bd4a40ceaad

<img width="649" alt="Image" src="https://github.com/user-attachments/assets/8d5e683f-dbb7-4338-90b9-37e92708d13a" />

**Comment by MuShangCC** [comment]
单独 import torch 或者 import paddle 都没问题吗？基于什么需求下要同时使用torch和paddle呢?
